modules/programs/aerospace.nix

```diff
@@ -81,24 +94,7 @@ in
       };
     };
 
-    extraConfig = mkOption {
```

Comment:
Sorry, missed before. But we should have a `mkRemovedOptionModule` that explains why it was removed.